content/pages/burials-and-memorials/survivor-and-dependent-benefits.md

```diff
@@ -29,3 +34,11 @@ relatedlinks:
 We give the surviving spouse, children, and parents of deceased Servicemembers and Veterans a range of benefits. These may include tax-free monetary benefits and help paying for the deceased's funeral service.
 
 </div>
+
+### Ready to apply?
```

Comment:
Is the intentional? If it didn't have an apply link before, do we want it to have the same treatment as the main burials page, where we hid the apply button but show a saved application (if it exists)?

Reply:
Good question, @peggygannon echoed your idea above, this was probably the intention of the designs in the original issue as well. This has been changed and the screenshot updated. Similar change will be made for pension and hca landing pages.